cmd/loadTests.go

```diff
@@ -151,106 +189,131 @@ func setup(cmd *cobra.Command, args []string) {
+			ComponentContainerImage := fmt.Sprintf("quay.io/%s/test-images:%s-%s", utils.GetQuayIOOrganization(), username, strings.Replace(uuid.New().String(), "-", "", -1))
 			component, err := framework.AsKubeAdmin.HasController.CreateComponent(
 				ApplicationName,
 				ComponentName,
-				username,
+				usernamespace,
 				QuarkusDevfileSource,
 				"",
 				"",
 				ComponentContainerImage,
 				"",
 				true,
 			)
-			if component.Name != ComponentName {
-				klog.Fatalf("Component Name Does not Match: %v", err)
-			}
 			if err != nil {
-				klog.Fatalf("Problem Creating the Component: %v", err)
+				logError(6, fmt.Sprintf("Failed to create the Component %s: %v", ComponentName, err))
+				atomic.AddInt32(&FailedResourceCreations, 1)
+				continue
 			}
-			if ResourcesBar.Current()%userBatches == 0 {
-				for i := ResourcesBar.Current() - userBatches + 1; i < ResourcesBar.Current(); i++ {
+			if component.Name != ComponentName {
+				logError(7, fmt.Sprintf("Component Name Does not Match: %v", err))
+				atomic.AddInt32(&FailedResourceCreations, 1)
+				continue
+			}
+			if userIndex%userBatches == 0 {
+				for i := userIndex - userBatches + 1; i <= userIndex; i++ {
 					time.Sleep(time.Second * 1)
 					// Todo Add validation after each batch
 				}
 			}
 			ResourceCreationTime := time.Since(startTime)
 			AverageResourceCreationTimePerUser += ResourceCreationTime
+			chPipelines <- userIndex
+			ResourcesBar.Incr()
 		}
+		close(chPipelines)
 		wg.Done()
 	}()
 	if waitPipelines {
 		PipelinesBar := uip.AddBar(numberOfUsers).AppendCompleted().PrependFunc(func(b *uiprogress.Bar) string {
-			return strutil.PadLeft(fmt.Sprintf("Pipelines running (%d/%d)", b.Current(), numberOfUsers), userBatches, ' ')
+			return strutil.PadLeft(fmt.Sprintf("Waiting for pipelines to finish (%d/%d) [%d failed]", b.Current(), numberOfUsers, FailedPipelineRuns), userBatches, ' ')
 		})
 		go func() {
-			for PipelinesBar.Incr() {
-				username := fmt.Sprintf("%s-%04d", usernamePrefix, ResourcesBar.Current())
+			for userIndex := range chPipelines {
+				username := fmt.Sprintf("%s-%04d", usernamePrefix, userIndex)
+				usernamespace := fmt.Sprintf("%s-tenant", username)
 				ComponentName := fmt.Sprintf("%s-component", username)
 				ApplicationName := fmt.Sprintf("%s-app", username)
 				DefaultRetryInterval := time.Millisecond * 200
 				DefaultTimeout := time.Minute * 17
 				error := k8swait.Poll(DefaultRetryInterval, DefaultTimeout, func() (done bool, err error) {
-					pipelineRun, err := framework.AsKubeAdmin.HasController.GetComponentPipelineRun(ComponentName, ApplicationName, username, "")
+					pipelineRun, err := framework.AsKubeAdmin.HasController.GetComponentPipelineRun(ComponentName, ApplicationName, usernamespace, "")
 					if err != nil {
-						return false, err
+						//klog.Warningf("Error when getting component (%s/%s) pipeline run: %+v", ApplicationName, ComponentName, err)
```

Comment:
Are the comments on lines 303 and 309 supposed?

Reply:
nope, I forgot to remove them... thx

Reply:
fixed